### PR TITLE
add include_package_data to setup.cfg in npe2 convert

### PR DESCRIPTION
**Pull request by tlambert03**

### CODE REVIEW

**Review by nclack — APPROVED** [review]
thanks!

**Comment by nclack** [comment]
ack, just saw the test failure.  I guess `options` needs to be added.

File: npe2/_from_npe1.py [hunk 1]
```diff
@@ -490,6 +490,7 @@ def _write_new_setup_cfg_ep(info: PluginPackage, mf_name: str):
     if "options.package_data" not in p.sections():
         p.add_section("options.package_data")
     p.set("options.package_data", info.top_module, mf_name)
+    p.set("options", "include_package_data", "True")
```
Comment (nclack) [inline npe2/_from_npe1.py:493]:
```suggestion
if "options" not in p.sections():
        p.add_section("options")
p.set("options", "include_package_data", "True")
```

**Review by DragaDoncila — APPROVED** [review]
Looks good! Thanks @tlambert03

**Pull request merged**